packages/docusaurus-plugin-content-docs/src/lastUpdate.ts

```diff
@@ -31,7 +31,11 @@ export default function getFileLastUpdate(
 
   // Wrap in try/catch in case the shell commands fail (e.g. project doesn't use Git, etc).
   try {
-    // To differentiate between content change and file renaming / moving, use --summary
+    if (!shell.which('git')) {
+      console.log('Sorry, the docs plugin last update options require Git.');
```

Comment:
hmm wait.. will this log 100 times if we have 100 docs 😭

Reply:
Oops, good point. What's a good way to dedupe this? I could use a variable in the module.